pylearn2/termination_criteria/__init__.py

```diff
@@ -221,16 +221,16 @@ def initialize(self, model):
         if self._new_epochs:
             self._epochs_done = 0
```

Comment:
```new_epochs``` Was to allow counting from new training no matter how many epochs it was trained beforehand. If new_epochs is False, the model is trained while ```max_epochs > old_epochs + epochs_done```. Otherwise, old_epochs is reset to 0 and the model is trained while ```max_epochs > epochs_done```.